src/ucioption.cpp

```diff
@@ -79,9 +80,7 @@ void init(OptionsMap& o) {
   o["Syzygy50MoveRule"]      << Option(true);
   o["SyzygyProbeLimit"]      << Option(7, 0, 7);
   o["Use NNUE"]              << Option(true, on_use_NNUE);
-  // The default must follow the format nn-[SHA256 first 12 digits].nnue
-  // for the build process (profile-build and fishtest) to work.
-  o["EvalFile"]              << Option("nn-82215d0fd0df.nnue", on_eval_file);
+  o["EvalFile"]              << Option("nn-82215d0fd0df.nnue", on_eval_file);  // TODO : change to EvalFileDefaultName
```

Comment:
I think this can now be changed to EvalFileDefaultName as the CI should work.

Reply:
Done